src/types/Errors.ts

```diff
@@ -0,0 +1,9 @@
+export enum ErrorType {
+  UnableToLoadTodo = 'Unable to load todos',
+  TitleIsEmpty = 'Title should not be empty',
+  UnableToAddTodo = 'Unable to add a todo',
+  UnableToDeleteTodo = 'Unable to delete a todo',
+  UnableToUpdateTodo = 'Unable to update a todo',
```

Comment:
Names in enums have to be in upercase. I think 'DELETE' instead of 'UnableToDeleteTodo' should be better